packages/suite/src/components/wallet/CoinmarketLayout/components/Navigation/index.tsx

```diff
@@ -11,21 +23,53 @@ const Navigation = () => {
         { route: 'wallet-coinmarket-spend', title: 'TR_NAV_SPEND' },
     ] as const;
 
-    const routeName = useSelector(state => state.router.route?.name);
+    const { routeName, account, savingsProviders } = useSelector(state => ({
+        routeName: state.router.route?.name,
+        account: state.wallet.selectedAccount?.account,
+        savingsProviders: state.wallet.coinmarket.savings.savingsInfo?.savingsList?.providers,
+    }));
     const { goto } = useActions({
         goto: routerActions.goto,
     });
 
+    const showSavingsTab =
+        account &&
+        savingsProviders &&
+        savingsProviders.some(
+            savingsProvider =>
+                savingsProvider.isActive &&
+                savingsProvider.tradedCoins.includes(account.symbol.toUpperCase()),
+        );
+
     return (
         <WalletLayoutNavigation>
-            {items.map(({ route, title }) => (
-                <WalletLayoutNavLink
-                    key={route}
-                    title={title}
-                    active={routeName === route}
-                    onClick={() => goto(route, { preserveParams: true })}
-                />
-            ))}
+            <>
+                {items.map(({ route, title }) => (
+                    <WalletLayoutNavLink
+                        key={route}
+                        title={title}
+                        active={routeName === route}
+                        onClick={() => goto(route, { preserveParams: true })}
+                    />
+                ))}
+                {showSavingsTab && account && (
```

Comment:
`&& account` is already included in `showSavingsTab` condition 
```suggestion
                {showSavingsTab && (
```

Reply:
Fixed in 0f297ba